src/main/java/com/thevoxelbox/voxelsniper/snipe/Sniper.java

```diff
@@ -98,9 +99,9 @@ public boolean snipe(Action action, Material itemInHand, Block clickedBlock, Blo
             }
 
             SnipeData snipeData = sniperTool.getSnipeData();
+            Block targetBlock;
```

Comment:
All looks good, only small thing is that I don't really like these kinds of variable declarations. I feel like, working on code, I want to quickly see what is assigned to a variable. The IDE moves your curser to the declaration when you click a variable. Therefore I usually like it better if the declaration is as close to the assignment as possible.

Reply:
Hard to get around that when the variable is being set in two different ways